test/dynamo/test_trace_rules.py

```diff
@@ -78,6 +88,22 @@
     "torch._C._swap_tensor_impl",
     "torch._C._unsafe_reset_storage",
     "torch._dynamo.eval_frame.reset_code",
+    # "torch.jit.isinstance",
+    "torch._C.autocast_decrement_nesting",
+    "torch._C.autocast_increment_nesting",
+    "torch._C.clear_autocast_cache",
+    "torch._C.set_anomaly_enabled",
+    "torch._C.set_autocast_cache_enabled",
+    "torch._C.set_autocast_cpu_dtype",
+    "torch._C.set_autocast_cpu_enabled",
+    "torch._C.set_autocast_enabled",
+    "torch._C.set_autocast_gpu_dtype",
+    "torch._C.set_autocast_ipu_dtype",
+    "torch._C.set_autocast_ipu_enabled",
+    "torch._C.set_autocast_xla_dtype",
+    "torch._C.set_autocast_xla_enabled",
+    "torch.resize_as_",
+    "torch.resize_as_sparse_",
```

Comment:
We removed these functions from the ```torch_c_binding_in_graph_functions``` list in last PR #116365, but somehow the CI skipped the unit test ```test_torch_name_rule_map_updated``` because of flaky. I found failure when I run local test, next step I'll work with dev infra team to enable that unit test again.